### PR TITLE
feat: Support envd-server

### PR DESCRIPTION
**Pull request by gaocegege**
It is a significant change to envd. The PR is to:

- support `envd login`

Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

File: pkg/app/ssh.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	"github.com/cockroachdb/errors"
+	"github.com/urfave/cli/v2"
+
+	"github.com/tensorchord/envd/pkg/home"
+	"github.com/tensorchord/envd/pkg/ssh"
+	sshconfig "github.com/tensorchord/envd/pkg/ssh/config"
+)
+
+var CommandSSH = &cli.Command{
+	Name:     "ssh",
```
Comment (VoVAllen) [inline pkg/app/ssh.go:27]: Is this what we planned for `envd attach`? `attach` sounds better to me

**Review by VoVAllen — COMMENTED** [review]
LGTM as the initial poc.

**Review by VoVAllen — APPROVED** [review]

File: pkg/app/ssh.go (hunk 1 above)
Comment (gaocegege) [reply pkg/app/ssh.go:27]: I am not sure if we need this. the CLI UX design should be refined soon. Let's refine it later.

File: pkg/app/login.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+		IdentityToken: resp.IdentityToken,
+	}, true); err != nil {
+		return errors.Wrap(err, "failed to create the auth config")
+	}
+	fmt.Println(resp.Status)
+	return nil
+}
+
+func GetMD5Hash(text string) string {
+	hasher := md5.New()
```
Comment (kemingy) [inline pkg/app/login.go:76]: Is MD5 secure in this scenario?

Comment (gaocegege) [reply pkg/app/login.go:76]: This token is used as the unique identifier of the user. Thus it should work.

**Review by kemingy — APPROVED** [review]

**Pull request merged**